docs/source/guides/auth/index.rst

```diff
@@ -0,0 +1,56 @@
+.. _authGuide:
+
+Authentication and Authorization
+=======================================
```

Comment:
Doesn't it have to match the header text length?

Reply:
Or maybe it couldn't be less then the length of the text...

Reply:
> Or maybe it couldn't be less then the length of the text...

Yup, it has to be less than the length of the text